model_bakery/generators.py

```diff
@@ -70,6 +72,19 @@
     CIEmailField = None
     CITextField = None
 
+try:
+    from django.contrib.postgres.fields.ranges import DecimalRangeField
+except ImportError:
+    DecimalRangeField = None
+try:
+    from django.contrib.postgres.fields.ranges import (
+        IntegerRangeField,
+        BigIntegerRangeField,
+    )
+except ImportError:
```

Comment:
Why not have them in the same `try/except` block?

```suggestion
try:
    from django.contrib.postgres.fields.ranges import (
        BigIntegerRangeField,
        DecimalRangeField,
        IntegerRangeField,
    )
except ImportError:
    BigIntegerRangeField = None
    DecimalRangeField = None
    IntegerRangeField = None
```

Reply:
This is because `DecimalRangeField` came after the other range fields. It was [introduced in Django 2.2](https://docs.djangoproject.com/en/3.0/ref/contrib/postgres/fields/#decimalrangefield) to deprecate the `FloatRangeField`. So, if we place these imports together, a project using Django 2.1 won't be able to use `BigIntegerField`, for example, because of the failed import of `DecimalRangeField`.

Anyway, I'll add support for the `FloatRangeField` as well because people may want support if using Django 2.1 or 2.0

Reply:
Makes sense. Thanks!